lib/mayaUsd/render/vp2RenderDelegate/mesh.h

```diff
@@ -37,23 +46,26 @@ class HdVP2RenderDelegate;
     draw data from these shared data as needed.
 */
 struct HdVP2MeshSharedData {
```

Comment:
If this is going to be the mesh topology should we re-name the struct as well?

Reply:
I don't quite follow what you mean here. If I guess it correctly, it is the doc change that leads to your question?

There is still primvar data in the struct - i.e. _primvarSourceMap.

Reply:
Yes the documentation change cause the question.